### PR TITLE
Fix from new review

### PR DESCRIPTION
**Pull request by duane-mcmahon**
updated classes in original adhoc tickets module to the new style and coding standards most recently adopted.

### CODE REVIEW

**Comment by duane-mcmahon** [comment]
As per the previous comment, now merged. please refer to the Phase Status document; i committed and pushed some accidental style changes applied to class that were not my responsibility.

File: src/bcccp/carpark/paystation/PaystationController.java [hunk 1]
```diff
@@ -47,9 +47,9 @@ public void ticketPaid() {
 
   @Override
   public void ticketTaken() {
```
Comment (WBGroupR) [inline src/bcccp/carpark/paystation/PaystationController.java:49]: Google Style Guide 4.2 Block indentation should be +2 spaces

**Comment by WBGroupR** [comment]
But I haven’t had a chance to fully review the code...

**Comment by duane-mcmahon** [comment]
We didn't need it to be reviewed if it was actioning a refactored review. If there are still issues by all means point it out.

File: src/bcccp/carpark/paystation/PaystationController.java (hunk 1 above)
Comment (duane-mcmahon) [reply src/bcccp/carpark/paystation/PaystationController.java:49]: yeah i'm not actually responsible for paystation controller, that is chris' class.

Comment (duane-mcmahon) [reply src/bcccp/carpark/paystation/PaystationController.java:49]: something that happened accidentally was when  i commited and tried to push my changes the style checker blocked the push cos of all the style errors on all the other classes that I wasn't responsible for. Accidentally i may pressed "Format Code' on a class or two that was not my own before realising

**Comment by WBGroupR** [comment]
OK, sorry, yeah , I recall now. I just read about GitHub and it looks like there is some .editorconfig file that can be set in the repository in which the indent settings can be changed. Not suggesting that we do that at this late stage, but we just have to remember that GitHub doesn’t necessarily show the code style as its written in IntelliJ - might add that as another issue/comment

W

**Comment by WBGroupR** [comment]
ok, no worries

**Comment by duane-mcmahon** [comment]
"OK, sorry, yeah , I recall now. I just read about GitHub and it looks like there is some .editorconfig file that can be set in the repository in which the indent settings can be changed. "  yeah  i am still learning github mate lolz. i must remember in the final commit/push dialog to check only the specific files i want in the branch to be pushed not 'everything and the kitchen sink'.

**Pull request merged**